features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql

```diff
@@ -1663,6 +1663,7 @@ CREATE TABLE AM_SUBSCRIPTION (
     UPDATED_BY VARCHAR(100),
     UPDATED_TIME DATETIME,
     UUID VARCHAR(256),
+    VERSION_RANGE VARCHAR(100) NULL,
```

Comment:
100 isn't too much ? 
what would the expected values for this column ?

Reply:
This column is supposed to have a regex for the version range. Let's have varchar(100) for worst case.